moderator-tools/ReviewQueueNotification.js

```diff
@@ -1,24 +1,24 @@
 /** @preserve
 // ==UserScript==
 // @name Review Queue Notification
-// @author Malachi Edited Simon Forsberg Created
-// @description Shows a desktop notification when there review items in the queue.
-// @namespace https://github.com/Zomis/SE-Scripts
+// @author Malachi with help from Simon Forsberg
```

Comment:
Might I suggest removing @author tags? See [this link](http://stackoverflow.com/a/17271433/3626537). Also I think the proper place for a PayPal link would be in the README.md file in the repository, not in the code itself, as it doesn't say anything useful about the code.